packages/moveable/src/overlay/index.js

```diff
@@ -19,11 +19,11 @@
  */
 import styled from 'styled-components';
 import { forwardRef, useContext, createPortal } from '@googleforcreators/react';
+import { pointerEventsCss } from '@googleforcreators/design-system';
```

Comment:
At first glance it looks like this import and the SVG import in`moveStyle.js` are the only places where this now depends on the design system.

I wonder if we could/should inline those usages so we could avoid this dependency? And if that is any useful?

Reply:
That's a good idea to remove design-system dependency.

Reply:
Looks like that made a difference of another 700b, so the bundle size is now reduced by 1.4kB in total 👍